src/main/java/org/jenkinsci/plugins/workflow/support/DefaultStepContext.java

```diff
@@ -194,7 +195,8 @@ public PasswordParameterEnvironmentExpander(Run<?, ?> run) {
             ParametersAction action = run.getAction(ParametersAction.class);
             if (action != null) {
                 passwordParameterVariables = action.getParameters().stream()
-                        .filter(PasswordParameterValue.class::isInstance)
+                        .filter(e -> PasswordParameterValue.class.isInstance(e)
```

Comment:
nit: I was using `Class.isInstance` to avoid a lambda, but since we have the lambda now anyway it might be clearer to write `e instanceof PasswordParameterValue` instead.